Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -240,10 +240,14 @@ def IsLimit (o : Ordinal) : Prop :=
   o ≠ 0 ∧ ∀ a < o, succ a < o
 #align ordinal.is_limit Ordinal.IsLimit
 
+theorem IsLimit.isSuccLimit {o} (h : IsLimit o) : IsSuccLimit o := isSuccLimit_iff_succ_lt.mpr h.2
```

Comment:
There is an asymmetry between the definitions
```
Ordinal.IsLimit o = (o ≠ 0 ∧ ∀ a < o, Order.succ a < o)
Cardinal.IsLimit c = (c ≠ 0 ∧ Order.IsSuccLimit c)
```
but it's probably out of scope of this PR to align them.